security/experimental_authenticators.rst

```diff
@@ -494,15 +494,15 @@ The following credential classes are supported by default:
         use Symfony\Component\Security\Http\Authenticator\Passport\Credentials\PasswordCredentials;
 
         // ...
-        return new Passport($user, new PasswordCredentials($plaintextPassword));
+        return new Passport(new UserBadge($email), new PasswordCredentials($plaintextPassword));
```

Comment:
Can you please add a use statement? Thanks

Reply:
I don't think it's needed. This is a really small code snippet inside a definition list (so I prefer to keep it as minimal as possible). The use statement is present in all code examples prior to this change (but those were already correct).